_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation-filter-popin/smart-shopping-campaign-creation-popin-recap.vue

```diff
@@ -8,13 +8,13 @@
   >
     <dl class="w-100 d-inline-block">
       <dt class="font-weight-600">
-        {{ $t('smartShoppingCampaignCreation.inputNameLabel') }}
+        {{ $t("smartShoppingCampaignCreation.inputNameLabel") }}
```

Comment:
You may have to change something in the configuration of your IDE, there are too many changes like this one.